sentences/sv/light_HassLightSet.yaml

```diff
@@ -13,9 +13,9 @@ intents:
           name: "all"
 
       - sentences:
-          - "(färga | ändra | byt) <name> [till] {color}"
+          - "<färga> <name> [till] {color}"
 
       - sentences:
-          - "{color} ljus i <area>"
+          - "[<change>] [till ] {color} ljus i <area>"
```

Comment:
`<change>` doesn't exist anymore, shouldn't it be changed to `färga`?

Reply:
Ah, dang, missed it, will fix tomorrow!

Reply:
I think we should be good to go now.